module/zstd/zstd.c

```diff
@@ -664,6 +664,6 @@ EXPORT_SYMBOL(zstd_get_level);
 ZFS_MODULE_DESCRIPTION("ZSTD Compression for ZFS");
 ZFS_MODULE_LICENSE(ZFS_META_LICENSE);
 // FIXME
-//ZFS_MODULE_VERSION("1.4.4");
+// ZFS_MODULE_VERSION("1.4.4");
 MODULE_VERSION("1.4.4");
```

Comment:
we need to decide which version we want here... zfs as all other modules or zstd?

Reply:
@c0d3z3r0 
The used ZSTD lib doesn't have to be the same version as the ZSTD ZFS module...

For example:
If a new ZSTD version 1.4.5 comes out which has no implications for the ZFS ZSTD module, only the version of the ZSTD lib would change.

Reply:
(and on the other hand: the ZSTD module might change but the LIB stays the same.)

Reply:
Or do you mean something else?

Reply:
MODULE_VERSION defines the version of the kernel module; all other ZFS modules use the ZFS version. IMHO we should do the same here

Reply:
As we are "just another module" we should indeed stay consitent

Reply:
ok, done